src/modules/recipe/eventWithData/__tests__/eventWithData.test.js

```diff
@@ -0,0 +1,68 @@
+import { createElement } from 'lwc';
+import EventWithData from 'recipe/eventWithData';
+import { registerLdsTestWireAdapter } from '@salesforce/wire-service-jest-util';
+import getContactList from 'data/wireGetContactListProvider';
+
+const CONTACTS = [
+    {
+        Id: '0031700000pJRRSAA4',
+        Name: 'Amy Taylor',
+        Title: 'VP of Engineering',
+        Phone: '4152568563',
+        Email: 'amy@demo.net',
+        Picture: '/resources/images/demo/amy_taylor.jpg'
+    }
+];
+
+const getContactListWireAdapter = registerLdsTestWireAdapter(getContactList);
+
+describe('recipe-event-with-data', () => {
```

Comment:
I'd like to see overall alignment between the tests in `lwc-recipes` and `lwc-recipes-oss`.

Reply:
done.